scheduler/cloudformation/template.go

```diff
@@ -167,6 +168,7 @@ func (t *EmpireTemplate) Build(app *scheduler.App) (*troposphere.Template, error
 		default:
 			service := t.addService(tmpl, app, p)
 			serviceMappings = append(serviceMappings, Join("=", p.Type, Ref(service)))
+			deploymentMappings = append(deploymentMappings, Join("=", p.Type, GetAtt(service, "DeploymentId")))
```

Comment:
We currently have support for using the standard `AWS::ECS::Service` resource, which wouldn't have the `DeploymentId` attribute. We should either make this conditional, or remove support for the standard ecs service, which I think would be fine.